analyzer/Dockerfile.jooola

```diff
@@ -0,0 +1,39 @@
+FROM python:3-buster as analyzer-build
+
+WORKDIR /build
+
+# How to keep this in sync across the many Dockerfiles ?
+ADD https://raw.githubusercontent.com/libretime/libretime/master/tools/packages.py /tmp/packages.py
+COPY packages.ini /tmp/
+# TODO: Rename packages.ini common section to python
+# TODO: Properly handle python3 setup provided by the initial docker image and
+#       our dependencies automagically installing python from apt.
+#       This mainly has to do the python Gstreamer dependencies.
+RUN python3 /tmp/packages.py \
+    --format=line buster \
+    --exclude=common \
+    /tmp/packages.ini > packages.list
+
+COPY . .
```

Comment:
This will result in dependencies being downloaded and installed every time there is a code update, which makes developing using the docker images slow

Reply:
Gotcha. In that case would a requirement.txt not be welcomed here ?

Reply:
Definitely. Though a way of doing it without maintaining two lists of dependencies would be great

Reply:
Maybe https://github.com/bndr/pipreqs cal help with that, generate it on fly inside docker or as make dependency ?

Reply:
We have a working way of generating the requirements.txt already in the Dockerfile - https://github.com/LibreTime/libretime/pull/1471/files#diff-cfe7fb4506860afbb02571526ef58c47331b85e2625ae9ee3c403e4817ee9853R12-R22

Reply:
I don't like this piece of code. Not flexible enough and we need to duplicate this in few apps.
Any deviation would require extra change. For instance the api has some extra require that aren't 'dev'.

Reply:
Solving #1154 can help for this. Copying the pyproject + lock file will make dependency installation easier for the docker build, and in some other areas too.